package.json

```diff
@@ -5,8 +5,8 @@
   "description": "Typescript Typings for Sequelize",
   "typings": "index.d.ts",
   "dependencies": {
-    "@types/bluebird": "^3.5.5",
-    "@types/validator": "types/validator#83990944fcc53bcb4b05b347c3cdfcb7aa05b3c7"
```

Comment:
there must have been a reason why we used `types/validator`

Reply:
I figured as much but couldn't find any issues, the typings look mostly identical to me, `@types` are more precise/up to date however.

Reply:
Actually, they don't even work with the current codebase.

``` "validator": "registry:npm/validator#5.7.0+20160912014851"```